tensorflow_graphics/rendering/camera/perspective.py

```diff
@@ -266,27 +273,36 @@ def matrix_from_intrinsics(focal, principal_point, name=None):
                                [focal, principal_point]):
     focal = tf.convert_to_tensor(value=focal)
     principal_point = tf.convert_to_tensor(value=principal_point)
+    skew = tf.convert_to_tensor(value=skew)
 
-    shape.check_static(
-        tensor=focal, tensor_name="focal", has_dim_equals=(-1, 2))
+    shape.check_static(tensor=focal, tensor_name="focal", has_dim_equals=(-1, 2))
     shape.check_static(
         tensor=principal_point,
         tensor_name="principal_point",
-        has_dim_equals=(-1, 2))
+        has_dim_equals=(-1, 2),
+    )
+    shape.check_static(
+        tensor=skew,
+        tensor_name="skew",
+        has_dim_equals=(-1, 1),
+    )
     shape.compare_batch_dimensions(
```

Comment:
Sorry for not catching this earlier, but I think setting a default value of (0.0,) for skew and then raising an error through `compare_batch_dimensions` if the shapes don't match will break existing code that passes additional batch dimensions for `focal` and `principal_point` (right?). I think this function needs to use the `broadcast_to` style of https://github.com/tensorflow/graphics/blob/3dd63357f54f42ad60c6dc0daeb3e1739f7863be/tensorflow_graphics/geometry/transformation/rotation_matrix_3d.py#L398
This behavior really should be treated more consistently in tensorflow_graphics as a whole, but that's another issue...

Reply:
@fcole Will you please elaborate?

Reply:
@ghosalsattam what happens if you didn't change the test parameters on this line here: https://github.com/tensorflow/graphics/pull/406/files#r459409028 ? Is there an error?

Reply:
@fcole @fulkast What should be done now? Shall we leave the code at the current state in this regard or add broadcast_to/remove compare_batch_dimensions or shall we add broadcast_to?

Reply:
Sorry for the delay. Could you check what happens if you leave `skew=0.0` as the default argument, remove the shape checking code entirely and try the following:

1. run the original test with no skew argument (to simulate backwards compatibility)
2. run the new test with skew set. 

If that all passes ok, then I lean towards just removing the checking code.